lib/screens/settings/settings_screen.dart

```diff
@@ -26,58 +26,36 @@ class SettingsScreen extends StatelessWidget {
       body: BlocProvider(
         create: (context) =>
             SettingsBloc(BlocProvider.of<AuthenticationBloc>(context))..add(const SetUpInitialValues()),
-        child: MultiBlocListener(
```

Comment:
we should never have `MultiBlocListener` to listen to the same bloc 'AuthenticationBloc'

You only use `MultiBlocListener` when you need to listen to the state in different blocs.